src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicCipherSuitesPolicyTests.cs

```diff
@@ -10,6 +10,7 @@ namespace System.Net.Quic.Tests
 {
     [Collection(nameof(DisableParallelization))]
     [ConditionalClass(typeof(QuicTestBase), nameof(QuicTestBase.IsSupported))]
+    [SkipOnCoreClr("MsQuic is timing sensitive and flaky under stress", ~RuntimeTestModes.RegularRun)]
```

Comment:
There's no way to make the tests resilient against timing issues? What specifically is the problem that leads to the flakiness?

Reply:
We suspect #55979 as root cause. There is logic in MsQuic to watch load and start refusing connection if CPU utilization is too high. For normal tests runs we disabled parallel runs to avoid it but the stress may be still be too much.

To start with, we may disable stress only on ARM. It seems like it failed on ARM64 as well but it is hard to tell much as most of the links leads to nowhere now ;(